types/express/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "dependencies": {
+    "@types/body-parser": "*",
+    "@types/express-serve-static-core": "^4.17.18",
```

Comment:
Remove the others and add this one here: https://github.com/microsoft/DefinitelyTyped-tools/blob/master/packages/definitions-parser/allowedPackageJsonDependencies.txt

Reply:
I was thinking that as well, glad I got the second opinion. Thanks!